pages/content/amp-dev/documentation/guides-and-tutorials/integrate/story-interactive.md

```diff
@@ -0,0 +1,71 @@
+---
+$title: Add Interactivity support to Web Story tools
+$order: 4
+description: 'The [amp-story-interactive](https://amp.dev/documentation/components/amp-story-interactive/?format=stories) component provides various imersive elements that can be integrated into tools. Given the wide variety of options available to customize the elements, we have detailed what tools can do to have a seamless integration into the story creation workflow'
+formats:
+  - stories 
+---
+
+Interactive experiences provide a first-class support for user interactivity in Web Stories, and are designed with customization in mind. However, all the customization options can be overwhelming for users designing stories with interactive elements. We will be covering ways tools can integrate the interactive experiences into their existing workflows, and best practices to take into account.
```

Comment:
I thought the goal would be: "We know it's hard to render an actual Web Story for your Story creation flow preview, so here are some tips on how to integrate interactive components to your preview / WYSIWYG editor". The guidance on how things should look (ie: visual thresold, how to upload images...) will vary too much from one creation tool to the other.
Linking to all existing CSS variables or customization options is useful though.

1. How to integrate the polls into the preview/WYSIWYG editor
2. What are all the customization options

Sizing and all are already documented somewhere else this is just a duplicate